prow/config.yaml

```diff
@@ -4810,47 +4810,117 @@ periodics:
     - image: gcr.io/k8s-testimages/kubekins-e2e:v20171201-f8fa79b46-master
       args:
       - --root=/go/src
-      - "--clean"
-      - "--git-cache=/root/.cache/git"
-      - "--job=$(JOB_NAME)"
```

Comment:
huh? what's going on with those flags here? is the deletion intended?

Reply:
This is copied from `ci-kubernetes-node-kubelet`, it seems that someone removed all the useless flags